src/main/java/pl/craftserve/radiation/Radiation.java

```diff
@@ -180,17 +172,22 @@ public void run() {
                     boolean showBossBar = event.shouldShowWarning();
                     boolean cancel = event.isCancelled();
 
+                    boolean contains = bossBar.getPlayers().contains(player);
+
                     if (!cancel) {
                         for (PotionEffect effect : effects) {
                             player.addPotionEffect(effect, true);
                         }
 
                         addAffectedPlayer(player, showBossBar);
-                        return;
                     }
 
                     if (showBossBar) {
                         addBossBar(player);
+
+                        if (!contains) {
+                            broadcastEscape(player);
+                        }
```

Comment:
The message shouldn't be broadcasted even when the event is canceled. This about `/vanish` or something.

Reply:
I've checked previous version - before any of my changes, even when player is op and has a vanish a message is still broadcasted. Are you sure that was about `/vanish`?

Reply:
Hmm, I have probably used `canSee` to broadcast messages, nevermind.

Reply:
https://github.com/Craftserve/CraftserveRadiation/blob/abe0018ee7be3e75a65bb3d3aca8c9dc26414a90/src/main/java/pl/craftserve/radiation/Radiation.java#L132-L136